channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -390,6 +391,7 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
                         _customHeaders = null;
                     }
                     _enableSendPromise = senderConfig.enableSendPromise;
+                    _enableIntTracking = !!config.enableIntEndpointsTracking;
```

Comment:
I think this should actually be in the CfgSync plugin and not in the sender, likewise with the setting of the "disabled" it should be initiated from the cfgSync extension itself when in attempts to create the connection.

Reply:
Moved everything to cfgSync. Since cfgSync didn't use the openXhr from the helper function, so add the disabledProperty tag inside the plugin